patches/minecraft/net/minecraft/tileentity/TileEntityFurnace.java.patch

```diff
@@ -10,6 +10,15 @@
                              }
                          }
                      }
+@@ -242,7 +241,7 @@
+ 
+                 if (this.func_145950_i() && this.func_145948_k())
+                 {
+-                    ++this.field_174906_k;
++                    if(net.minecraftforge.event.ForgeEventFactory.onItemAttemptSmelt(field_145957_n, this.field_174906_k, this.field_174905_l))++this.field_174906_k;
+ 
+                     if (this.field_174906_k == this.field_174905_l)
```

Comment:
This is fired every tick.. this seems a bit excessive. It should be fired once and left to run it's cycle.